website/docs/cloud-docs/workspaces/settings/deletion.mdx

```diff
@@ -25,15 +25,33 @@ Before queuing a destroy plan, enable the **Allow destroy plans** toggle setting
 @include 'tfc-package-callouts/ephemeral-workspaces.mdx'
 <!-- END: TFC:only name:pnp-callout -->
 
-Automated destroy runs are available as part of the ephemeral workspaces feature.
+You can schedule a run to destroy all resources in a workspace at a specific
+date and time. This ephemeral workspace option reduces your infrastructure
+spend by helping you automatically destroy resources that you only need
+temporarily, such as development environments.
 
-Some use cases may benefit from allowing Terraform Cloud to schedule future infrastructure deletion automatically. For example, you can automatically destroy a workspace's resources used solely in a development environment after a certain period of time to save on cost.
+If you perform a destroy operation in the workspace before the scheduled auto
```

Comment:
This is my bad, the way it should have been written before is more like 

```This setting is automatically unset whenever Terraform Cloud performs an auto destroy run```

So technically not any destroy runs, only the automatic ones will unset this field. The main context here is that users may not anticipate that the setting will automatically be updated once completed, so we wanted to let them know in some way.

Reply:
thanks! i'll clarify